packages/client/modules/teamDashboard/components/TeamArchive/TeamArchive.tsx

```diff
@@ -192,16 +204,17 @@ const TeamArchive = (props: Props) => {
   const getIndex = (columnIndex: number, rowIndex: number) => {
     return columnCount * rowIndex + columnIndex
   }
-  const isRowLoaded = ({index}) => index < edges.length
+  const isRowLoaded = ({index}) => index < edges.length || index === noMoreMsgIndex
   const maybeLoadMore = () => {
-    if (!hasNext || isLoadingNext) return
-    loadNext(columnCount * 10)
+    if (!hasNext || isLoadingNext) return Promise.resolve()
+    return new Promise<void>((resolve, reject) => {
+      loadNext(columnCount * 10, {onComplete: (err) => (err ? reject(err) : resolve())})
+    })
   }
   const [cellCache] = useState(
     () =>
       new CellMeasurerCache({
-        defaultHeight: 182,
-        minHeight: 106,
+        defaultHeight: 70,
```

Comment:
oh that's a tricky one

When the row height is calculated by the lib at https://github.com/bvaughn/react-virtualized/blob/master/source/CellMeasurer/CellMeasurerCache.js#L217  it iterates through all possible columns. 
In the case of final row (where the card is rendered  with “no more” message) the lib fallback to `defaultHeight` at https://github.com/bvaughn/react-virtualized/blob/master/source/CellMeasurer/CellMeasurerCache.js#L149 for missed cells. 

This causes that the card with “no more” message visually has excessive space at the bottom (due to row height = defaultHeight).

<img width="1421" alt="Screenshot 2022-04-04 at 13 26 33" src="https://user-images.githubusercontent.com/790487/161525476-6d07c532-ceb6-4754-9056-d2cea8308526.png">

As a workaround, we could render invisible placeholders with some min-height emulating cards for holes. But we still need to get rid of `minHeight` in CellMeasurerCache config, because it leads to the same situation with excessive space at the bottom (but to a lesser extent).

Otherwise we could keep current changes (think about `defaultHeight
` only as fallback for holes) because the lib always measure real size of the cell (min/max sizes of which can be controlled by the card HTML/CSS layout itself anyway).

Reply:
Fair enough. We could explicitly set the height of holes to 0, but your solution works good enough